src/element-style-observer.js

```diff
@@ -234,10 +223,19 @@ export default class ElementStyleObserver {
 
 		properties = [...new Set(properties)]; // Dedupe
 
-		// Only add properties not already present and not excluded
+		let supportsAllowDiscrete = CSS.supports("transition-behavior", "allow-discrete");
```

Comment:
This is not going to change. Just set it as a `const` on the outside. I would also suggest setting it to `" allow-discrete"` if true and `''` if false, then you can just append it when you need it instead of having to use a conditional.

Reply:
Makes sense. Fixed!